kotlinx-coroutines-core/common/src/channels/AbstractChannel.kt

```diff
@@ -48,7 +48,8 @@ internal abstract class AbstractSendChannel<E> : SendChannel<E> {
             val receive = takeFirstReceiveOrPeekClosed() ?: return OFFER_FAILED
             val token = receive.tryResumeReceive(element, null)
```

Comment:
It looks like  `tryResumeReceive` can return either `Symbol` or null, it's time to update the signature